python/docs/source/reference/pyspark.sql/dataframe.rst

```diff
@@ -109,6 +109,7 @@ DataFrame
     DataFrame.tail
     DataFrame.take
     DataFrame.to
+    DataFrame.toArrowTable
```

Comment:
One last thing. Should we name it `toArrow` just to match with others like `toPandas`, `toJSON`, etc.? I don't have a strong preference though.

Reply:
Yeah, let's probably name it `toArrow` just to match with others, e.g., https://arrow.apache.org/docs/python/generated/pyarrow.Table.html#pyarrow.Table.to_pandas too

Reply:
Either name is fine by me, but @grundprinzip  commented that he prefers `toArrowTable ` https://github.com/apache/spark/pull/45481#discussion_r1592074090

Reply:
Let's go with toArrow. I think this kind of naming is more common, e.g., https://datatable.readthedocs.io/en/latest/api/frame/to_arrow.html https://docs.pola.rs/py-polars/html/reference/dataframe/api/polars.DataFrame.to_arrow.html

Reply:
Sounds good. Done.